core/app/models/spree/order.rb

```diff
@@ -400,7 +400,7 @@ def deliver_order_confirmation_email
 
     # Helper methods for checkout steps
     def paid?
-      payment_state == 'paid' || payment_state == 'credit_owed'
+      %w(paid credit_owed).include?(payment_state)
```

Comment:
I wounder if we should move the array literal to a (deep frozen) constant of type `Set` this would be more intention revealing. Set not because of performance, but to use a datastructure that does not allow duplicates.

```
PAYED_STATES = IceNine.deep_freeze(%w[paid credit_owed].to_set)
```

Deep frozen to prevent any global state mutation by accident.